src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectFactory.cs

```diff
@@ -31,33 +31,35 @@ internal sealed class VisualStudioProjectFactory
             _hostDiagnosticUpdateSource = hostDiagnosticUpdateSource;
         }
 
-        public VisualStudioProject CreateAndAddToWorkspace(string projectUniqueName, string language)
+        public VisualStudioProject CreateAndAddToWorkspace(string projectSystemName, string language)
```

Comment:
Does this mean "project-system name" of "project system-name"?

Reply:
The former.